docs/about/NussknackerEnterprise.md

```diff
@@ -9,13 +9,13 @@ Available features and services:
 - ML Enhanced Decisions - a component that enriches your data with insights inferred from ML models
 - Decision Webhooks - a subscription mechanism that allows performing actions in external systems based on the 
 results of decision scenarios
-- Dedicated bug fix support
+- deployment of Flink engine scenarios to Ververica cloud
```

Comment:
1. Begin the sentence from a capital letter
2. "engine" is necessary here? Sounds long
3. In order to focus on the most important thing and make this less verbose I would start with "Ververica cloud ..." e.g "Ververica cloud integration" or "Ververica cloud scenarios deployment". WDYT?